### PR TITLE
Edits focused on improving clarity of message of best practices lesson.

### PR DESCRIPTION
**Pull request by BajczA475**
These edits focused primarily on making word choice more concrete and improving the learning objectives of this lesson. There were a number of instances where very vague pronouns like "it" were being used in place of more concrete nouns, and this can be really confusing. It's better to use specific but accessible nouns. I also fixed a couple of typos and other inconsistencies in here. The biggest thing, though, was changing the learning objectives--here, I tried to match the learning objectives with the exercises being given and make their connection to Bloom's taxonomy more obvious.

### CODE REVIEW

File: 06-best-practices-R.Rmd [hunk 1]
```diff
@@ -69,28 +71,27 @@ rather than:
...
+11. Don't save a session history (the default option in R, when it asks if you want an `RData` file). Instead, start in a clean environment so that older objects don't remain in your environment any longer than they need to. If that happens, it can lead to unexpected results.
 
-12. Where possible keep track of `sessionInfo()` somewhere in your project folder. Session information is invaluable since it captures all of the packages used in the current project. If a newer version of a project changes the way a function behaves, you can always go back and reinstall the version that worked (Note: At least on CRAN all older versions of packages are permanently archived).
+12. Wherever possible, keep track of `sessionInfo()` somewhere in your project folder. Session information is invaluable because it captures all of the packages used in the current project. If a newer version of a package changes the way a function behaves, you can always go back and reinstall the version that worked (Note: At least on CRAN, all older versions of packages are permanently archived).
```
Comment (BajczA475) [inline 06-best-practices-R.Rmd]: There really ought to be a more formal definition of what "session information" is in this section.

**Comment by gvwilson** [comment]
Thanks for the PR - @hdashnow and @chendaniely do you like the re-wording?

**Comment by hdashnow** [comment]
I think these changes make the wording clearer.
@BajczA475 would be be willing to rebase this PR? Or I can do it if you prefer. Let me know.

**Comment by chendaniely** [comment]
I'm +1 for the re-wording. thanks @BajczA475

**Comment by BajczA475** [comment]
Hello @hdashnow and @chendaniely, I'm glad the edits were useful. I think the lesson is really great, and I think the emphasis on best practices is really smart. @hdashnow my git is still pretty fuzzy as of now--can you rebase my PR this time? Thanks!

**Comment by chendaniely** [comment]
should merge this after #200 since there are a lot of files that need to be merged first and might make conflict resolution easier...

**Comment by hdashnow** [comment]
I've rebased, and submitted the changes as a new PR: #201

Thanks @BajczA475!

**Pull request closed without merging**